dap_aria_mapping/analysis/app/pages/2_Change_Makers.py

```diff
@@ -18,6 +24,121 @@
     page_icon=icon
 )
 
+@st.cache_data(show_spinner = "Loading Data")
+def load_networks(dataset: str) -> nx.Graph:
+    """get the collaboration network for either patents or publications
+
+    Args:
+        dataset (str): "Academia" for publication network or "Industry" for patent network 
+
+    Returns:
+        nx.Graph: collaboration network
+    """
+    return get_collaboration_network(dataset)
+
+@st.cache_data(show_spinner = "Loading Data")
+def load_filter_data(dataset: str) -> Tuple[pl.DataFrame, List[str]]:
```

Comment:
I wonder if for some of these we should load all the data, in effect reducing the number of S3 calls (but increasing memory use)

Reply:
the "filter_data" here is just the data used to populate the filter, it's not actually the network data, but this is changed in 73